WcaOnRails/spec/factories/users.rb

```diff
@@ -16,15 +16,24 @@
     factory :admin do
       name "Mr. Admin"
       email "admin@worldcubeassociation.org"
-      software_team true
+      after(:create) do |user|
+        software_team = FactoryGirl.create(:team, friendly_id: 'software', name: 'Software Team', description: 'Does software')
```

Comment:
I think not. Remember that each test has its own context and things don't keep existing outside? Why would we create 2 admins?